### DISCUSSION TITLE
Print feature?

### DISCUSSION THREAD

**Discussion by FSNaval** [firefly-iii/firefly-iii#7338] [General]
Are there any plans for implementing a print function/possibility at the reports section? 

If someone wants to print a report for records keeping, it is not possible to do it with the current setup. Adding a print possibility could prove useful. What do you think?

**Comment by JC5** [comment]
Is there anything in particular you're missing? There's a standard print CSS thing that's pretty good.

**Reply by FSNaval** [reply]
I have used various similar programs in the past, and they all have a print option, which i used in order to have a hard copy of my yearly income vs savings accounts for tax purposes.

I could not find this print CSS you are referring to, i would very much appreciate providing me with a link or a guidance to this.

**Reply by JC5** [reply]
Ehhh. Press Ctrl-P.
